app.js

```diff
@@ -18,3 +18,130 @@ menuLink.forEach((link) => {
     HTML.classList.remove('newHTML');
   });
 });
+
+const popUpData = [{
+  id: 1,
+  Heading: 'TONIC',
+  imageUrl: 'imagesdesktop/tonic2-1.png',
+  paraText: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been theindustrys standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languageList: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'],
+  button1Text: 'see live',
+  button2Text: 'see source',
+},
+{
+  id: 2,
+  Heading: 'TONIC',
+  imageUrl: 'imagesdesktop/tonic2-2.png',
+  paraText: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been theindustrys standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languageList: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'],
+  button1Text: 'see live',
+  button2Text: 'see source',
+},
+{
+  id: 3,
+  Heading: 'TONIC',
+  imageUrl: 'imagesdesktop/tonic2-3.png',
+  paraText: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry Lorem Ipsum has been the industry s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been theindustrys standard dummy text ever since the 1500s,when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languageList: ['html', 'css', 'javascript', 'github', 'ruby', 'bootstrap'],
+  button1Text: 'see live',
+  button2Text: 'see source',
+},
+{
+  id: 4,
```

Comment:
[OPTIONAL]

- [ ] You should use custom text rather than `lorem ipsum` :blush: